tests/Currencies/CZK.php

```diff
@@ -15,4 +15,5 @@ class CZK extends Currency
     protected string $thousandsSeparator = '.';
     protected int $rounding = 2;
     protected string $suffix = ' Kč';
+    protected bool $trimTrailingDecimalZeros = false;
```

Comment:
Remove these (`protected bool $trimTrailingDecimalZeros = false;`) from all of the files where it's false. It shouldn't need to be explicitly defined.

So that's USD, EUR, and CZK.